### PR TITLE
3687 fix error message force feed

### LINKED ISSUE #3687: Update translation(s) for feed error message
I'd look into it right now, but it's almost midnight.

<hr>

I just get a pink error message with a link to go back to my log. Might that be a translation error? 
imagen

_Originally posted by @ffuentese in https://github.com/FreshRSS/FreshRSS/issues/3686#issuecomment-874336822_

### PR DESCRIPTION
**Pull request by math-GH**
Closes #3687

Changes proposed in this pull request:

- added `You can try force adding by appending <code>#force_feed</code> to the URL.` to all languages

How to test the feature manually:
see #3687

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

File: app/i18n/cz/feedback.php [hunk 1]
```diff
@@ -84,7 +84,7 @@
 			'cache_cleared' => '<em>%s</em> cache has been cleared',	// TODO - Translation
 			'deleted' => 'Kanál byl smazán',
 			'error' => 'Kanál nelze aktualizovat',
-			'internal_problem' => 'RSS kanál nelze přidat. Pro detaily <a href="%s">zkontrolujte logy FreshRSS</a>.',
+			'internal_problem' => 'RSS kanál nelze přidat. Pro detaily <a href="%s">zkontrolujte logy FreshRSS</a>. You can try force adding by appending <code>#force_feed</code> to the URL.',	// TODO - Translation
```
Comment (Frenzie) [inline app/i18n/cz/feedback.php]: I don't think this will quite work with the automated tools unfortunately, but the CI will say soon enough.

Comment (math-GH) [reply app/i18n/cz/feedback.php]: fixed it

**Comment by Frenzie** [comment]
@aledeg Do you have a suggestion as to what would be the best thing to do here?

**Comment by aledeg** [comment]
I am willing to help but I am not sure to understand what you are expecting me to do!

**Comment by Frenzie** [comment]
You know the most about the translation tools. I guess we're missing a dirty like in gettext. So the best thing to do might be to reset the strings completely?

**Comment by aledeg** [comment]
I still don't get it. Do you mean something to notice translator that the string has changed and needs to be translated again?
If so, adding the ''// TODO - Translation'' string at the end of it would do the trick I think. Not sure though.

**Comment by Frenzie** [comment]
> If so, adding the ''// TODO - Translation'' string at the end of it would do the trick I think. Not sure though.

It would, but the way it's set up then the tools will automatically remove the todo because the strings are different.

**Comment by aledeg** [comment]
I see what you mean. Either we keep it as-is for now and we are trying to be a bit more clever with the translation tool or we remove completely the translation.
What could also work is splitting the string in 2. This way, the translated one does not change and the other one needs to be translated.

**Comment by Frenzie** [comment]
That's an interesting alternative solution. I think just removing it is probably no big deal seeing how it's just one string; just a pity we can't easily do dirty atm. ^-^

**Comment by math-GH** [comment]
what about to set this phrase on the ignore list?

**Comment by Frenzie** [comment]
Only deleting it or adding dirty to the tools will have the desired effect.

**Comment by math-GH** [comment]
Please decide what to do here to finish this PR.

**Comment by Frenzie** [comment]
Please delete/reset it in the affected languages.

**Comment by math-GH** [comment]
what should I delete?

**Comment by Frenzie** [comment]
I mean reset it to `'internal_problem' => 'Source English string',	// TODO - Translation`.

**Comment by math-GH** [comment]
What about to set this phrases on the ignore list? Would it work?

**Comment by math-GH** [comment]
Anything left to do?

File: app/i18n/cz/feedback.php [hunk 2]
```diff
@@ -84,7 +84,7 @@
 			'cache_cleared' => '<em>%s</em> cache has been cleared',	// TODO - Translation
 			'deleted' => 'Kanál byl smazán',
 			'error' => 'Kanál nelze aktualizovat',
-			'internal_problem' => 'RSS kanál nelze přidat. Pro detaily <a href="%s">zkontrolujte logy FreshRSS</a>.',
+			'internal_problem' => 'The newsfeed could not be added. <a href=%s>Check FreshRSS logs</a> for details. You can try force adding by appending <code>#force_feed</code> to the URL.',
```
Comment (Frenzie) [inline app/i18n/cz/feedback.php:87]:
```suggestion
			'internal_problem' => 'The newsfeed could not be added. <a href="%s">Check FreshRSS logs</a> for details. You can try force adding by appending <code>#force_feed</code> to the URL.',	// TODO - Translation
```

**Comment by Frenzie** [comment]
I'd noticed the lack of todo but I overlooked that the CI didn't mention it due to a subtly changed string.

**Comment by math-GH** [comment]
> I'd noticed the lack of todo but I overlooked that the CI didn't mention it due to a subtly changed string.

How could I add the TODO via CLI? I thought it will be done automatically via the CLI

**Comment by Frenzie** [comment]
It will do it automatically if you use the correct string. ;-)

**Comment by math-GH** [comment]
better PR: #4145

**Pull request closed without merging**